lib/ansible/modules/cloud/cloudstack/cs_volume.py

```diff
@@ -98,7 +98,7 @@
     description:
       - State of the volume.
```

Comment:
~~~diff
    description:
      - State of the volume.
+    - The choices I(extracted) and I(uploaded) were added in version 2.8. 
~~~

Reply:
OK! with C() as these are option values ;-)